### PR TITLE
KDE.kate version 21.04.1

### PR DESCRIPTION
**Pull request by SZanko**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16522)

### CODE REVIEW

File: manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml:3]:
```suggestion
PackageIdentifier: KDE.Kate
```

File: manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
+PackageVersion: 21.04.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/lastSuccessfulBuild/artifact/kate-21.04.1-1302-windows-msvc2019_64-cl.exe
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml:8]:
```suggestion
  InstallerUrl: https://build-artifacts.kde.org/binary-factory/Kate_Release_win64/1302/kate-21.04.1-1302-windows-msvc2019_64-cl.exe
```

File: manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: KDE.Kate
```

File: manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
+PackageVersion: 21.04.1
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 21.04.1.1302
```

File: manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
+PackageVersion: 21.04.1
+PackageLocale: en-US
+Publisher: KDE
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml:6]:
```suggestion
Publisher: KDE e.V.
```
image

File: manifests/k/KDE/kate/21.04.1/KDE.kate.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.yaml:3]:
```suggestion
PackageIdentifier: KDE.Kate
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

File: manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
+PackageVersion: 21.04.1
```
Comment (ItzLevvie) [inline manifests/k/KDE/kate/21.04.1/KDE.kate.installer.yaml:4]:
```suggestion
PackageVersion: 21.04.1.1302
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to rename the folder from `kate` to `Kate` using the GitHub Desktop client on Windows

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to fix the capitalization (i.e. from `kate` to `Kate` in the folder names and the file names).

File: manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml (hunk 5 above)
Comment (SZanko) [reply manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml]: Other Packages from KDE have only KDE as Publisher and not KDE e.V.

Comment (ItzLevvie) [reply manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml]: Those will be fixed soon.

**Comment by SZanko** [comment]
> You'll also need to fix the capitalization (i.e. from `kate` to `Kate` in the folder names and the file names).

In my file explorer I have the foldername changed to `Kate` from `kate` however git doesn't seem to track this change if I run it says that the branch is up to date.

**Comment by ItzLevvie** [comment]
Do:
`git reset 593e17f7e17458bccba3fba6bcc3505bfdc00136`
then change the folder names and files, and then write a commit summary making sure that all of the files are ticked in the GitHub Desktop client
and then do:
`git push --force`

**Comment by ItzLevvie** [comment]
You missed the capitalization of this file `manifests/k/KDE/Kate/21.04.1.1302/KDE.kate.locale.en-US.yaml`.

It needs to be changed from `kate` to `Kate`.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**